redo.dev/src/pages/index.tsx

```diff
@@ -36,9 +38,42 @@ const Contents = () => {
         </ThemeProvider>
     )
 }
-
 export default () => {
     const { siteConfig } = useDocusaurusContext()
+    enum Template {
+        ts = "typescript",
+        js = "javascript",
+        html = "html"
+    }
+    const getEmbedOptions: any = () => {
+        return {
+            clickToLoad: false,
+            view: "preview",
+            height: "100%",
+            width: "100%"
+        }
+    }
+    const createStackblitzDemo = (
```

Comment:
DUDE THIS IS FUCKING AMAZING GITHUB INTEGRATES CI LINTING RESULTS WITH PRs AUTOMATICALLY!?!?!?!?

But all of the new stuff here was for testing and should be deleted I think?